src/WindowStateSaver.vala

```diff
@@ -125,7 +127,10 @@ public class Gala.WindowStateSaver : GLib.Object {
                 }
             }
 
-            window.move_resize_frame (false, last_x, last_y, last_width, last_height);
```

Comment:
Yeah I was wondering, shouldn't move_resize_frame with user op `false` constrain windows to be in the display area? A comment might be good to explain what's different here vs. the default constraints

Reply:
Done!